### PR TITLE
Image store, CV upload, Activity images, Cost images

### PR DESCRIPTION
**Pull request by Ruben-Brits**

### CODE REVIEW

File: libs/admin-console/feature/src/lib/admin-console.component.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { API } from "../../../../shared/api/api.service";
 import { Store } from "@ngxs/store";
+import { SetImgString } from 'libs/shared/ngxs/actions';
```
Comment (priyo5) [inline libs/admin-console/feature/src/lib/admin-console.component.ts:4]: Change to relative pathing

File: libs/parent-view-activity/feature/src/lib/parent-view-activity.component.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
 import { Store } from '@ngxs/store';
+import { SetImgString } from 'libs/shared/ngxs/actions';
```
Comment (priyo5) [inline libs/parent-view-activity/feature/src/lib/parent-view-activity.component.ts:4]: Change to relative pathing

File: Server/src/main/java/Database/TheAuPair/Services/StorageService.java [hunk 3]
```diff
@@ -42,18 +42,21 @@ public String uploadFile(MultipartFile file)
...
+      S3Object s3Object = s3.getObject(bucketName, fileName);
+      S3ObjectInputStream inputStream = s3Object.getObjectContent();
+      try
+      {
+        byte[] content = IOUtils.toByteArray(inputStream);
+        return content;
+      }
+      catch (IOException e)
+      {
+        //e.printStackTrace();
```
Comment (PxttytheThxtty) [inline Server/src/main/java/Database/TheAuPair/Services/StorageService.java:55]: Commented Out Code

File: libs/au-pair-cost/feature/src/lib/extra-costs-modal/extra-costs-modal.component.ts [hunk 4]
```diff
@@ -330,4 +338,31 @@ export class ExtraCostsModalComponent implements OnInit {
     });
     await toast.present();
   }
+
+  selectFile(event: any) {
+    console.log(this.costDetails)
```
Comment (PxttytheThxtty) [inline libs/au-pair-cost/feature/src/lib/extra-costs-modal/extra-costs-modal.component.ts:343]: Console log

File: libs/au-pair-profile/feature/src/lib/au-pair-profile.component.scss [hunk 5]
```diff
@@ -240,3 +240,31 @@
...
+    padding-top: 10px;
+    min-width: 120px;
+}
+
+
+.downloadCV{
+    height: 20px;
+    width: 20px;
+    cursor: pointer;
+}
```
Comment (PxttytheThxtty) [inline libs/au-pair-profile/feature/src/lib/au-pair-profile.component.scss:270]: No newline

File: libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.scss [hunk 6]
```diff
@@ -274,3 +274,51 @@ textarea{
...
+    text-align: center;
+}
+
+.cvRow{
+    display: flex;
+    justify-content: space-between;
+    vertical-align: middle;
+    align-items: center;
+    width: 100%;
+ }
```
Comment (PxttytheThxtty) [inline libs/edit-au-pair-profile/feature/src/lib/edit-au-pair-profile.component.scss:324]: No newline

File: libs/explore/feature/src/lib/expand-modal/expand-modal.component.scss [hunk 7]
```diff
@@ -121,3 +121,10 @@
         font-size: 12px !important;
     }
 }
+
+.downloadCV{
+    height: 20px;
+    width: 20px;
+    margin: 0px;
+    cursor: pointer;
+}
```
Comment (PxttytheThxtty) [inline libs/explore/feature/src/lib/expand-modal/expand-modal.component.scss:130]: No Newline

File: libs/shared/ngxs/actions.ts [hunk 8]
```diff
@@ -50,3 +50,8 @@ export class SetEmail{
     static readonly type = '[User] set email';
     constructor(public payload: string) {}
 }
+
+export class SetImgString{
+    static readonly type = '[User] set imgString';
+    constructor(public payload: string) {}
+}
```
Comment (PxttytheThxtty) [inline libs/shared/ngxs/actions.ts:57]: No newline

**Review by PxttytheThxtty — COMMENTED** [review]
You so iluvo

**Review by d-azmanov — DISMISSED** [review]
Nice work!

**Review by d-azmanov — APPROVED** [review]

**Review by PxttytheThxtty — APPROVED** [review]

**Pull request merged**